applications/StructuralMechanicsApplication/tests/cpp_tests/test_timoshenko_beams.cpp

```diff
@@ -20,7 +20,8 @@
 namespace Kratos::Testing
 {
 
-void CreateBeamModel2N(std::string TimoshenkoBeamElementName)
+template<SizeType TNNodes>
+void CreateBeamModel(std::string TimoshenkoBeamElementName, const std::vector<double>& rExpectedShearStress, const std::vector<double>& rExpectedBendingMoment)
```

Comment:
Since this function also does asserts, you could call it something like `ValidateBeamModel`. Also, let's make the element name const& to avoid a copy

Reply:
Ah, I see you just pushed the name change already 👍